src/Entity/CreditMemoUnit.php

```diff
@@ -40,11 +40,13 @@ public function getTaxesTotal(): int
 
     public function serialize(): string
     {
-        return json_encode([
+        $serialized = json_encode([
             'product_name' => $this->productName,
             'total' => $this->total,
             'taxes_total' => $this->taxesTotal,
         ]);
+
+        return ($serialized !== false) ? $serialized : '';
```

Comment:
I think we should throw an exception instead of silently returning an empty string.

Reply:
Agreed 👍